sklearn/multioutput.py

```diff
@@ -289,6 +288,12 @@ class MultiOutputRegressor(RegressorMixin, _MultiOutputEstimator):
 
         .. versionadded:: 1.0
 
+    See Also
+    --------
+    RegressorChain : Equivalent for regression.
+    MultiOutputClassifier : Classifies each output independently rather than
+        chaining.
```

Comment:
This is the docstring of `MultiOutputRegressor` so we need to change the description here.

```suggestion
    RegressorChain : A multi-label model that arranges regressions into a
        chain.
    MultiOutputClassifier : Classifies each output independently rather than
        chaining.
```